### PR TITLE
CON-3720 Ensure cert file is readable before trying to use it

### PR DESCRIPTION
**Pull request by apotterri**
Pin rake to fix the build, test against 4.9.

### CODE REVIEW

File: conjur-api.gemspec [hunk 1]
```diff
@@ -23,7 +23,7 @@ Gem::Specification.new do |gem|
   gem.add_dependency 'activesupport'
```
Comment (jvanderhoof) [inline conjur-api.gemspec]: Now that `activesupport` 5.0 is out, we need to bump the `gem.required_ruby_version = '>= 2.2.2'`

Comment (apotterri) [inline conjur-api.gemspec]: Why is that? The only requirement currently in the gemspec is >= 1.9, which 2.2.2 certainly meets. Presumably, activesupport's gemspec specifies whatever requirement it has.

Comment (jvanderhoof) [inline conjur-api.gemspec]: ActiveSupport 5 has a hard dependency on Ruby 2.2.2 => https://rubygems.org/gems/activesupport

Comment (apotterri) [inline conjur-api.gemspec]: Sure, but the API doesn't. It's up to activesupport's gemspec to specify which version of Ruby it requires.

**Pull request merged**